sdk/include/opentelemetry/sdk/logs/attribute_utils.h

```diff
@@ -0,0 +1,123 @@
+#pragma once
+
+#include <unordered_map>
+#include <vector>
+#include "opentelemetry/common/attribute_value.h"
+#include "opentelemetry/common/key_value_iterable_view.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace sdk
+{
+namespace logs
+{
+/**
+ * A counterpart to AttributeValue that makes sure a value is owned. This
+ * replaces all non-owning references with owned copies.
+ */
+using LogRecordAttributeValue = nostd::variant<bool,
+                                               int32_t,
```

Comment:
❤️ the fact that you use standard types here, i.e. `int32_t` - looks much cleaner than `int` we had in another class for traces. Good job!

Reply:
Can't take credit for this - these variants are already defined in `AttributeValue` :)